src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
The `alive` array is correctly defined as a static property of the `Animal` class, which is appropriate for tracking all instances of `Animal`.